truss/truss_handle.py

```diff
@@ -150,6 +150,51 @@ def get_docker_image(self, labels: Dict):
         """[Deprecated] Do not use."""
         return _docker_image_from_labels(labels)
 
+    @proxy_to_shadow_if_scattered
+    def run_python_script(
+        self,
+        script_path: Path,
+        build_dir: Optional[Path] = None,
+    ):
+        image = self.build_serving_docker_image(build_dir=build_dir)
+        secrets_mount_dir_path = _prepare_secrets_mount_dir()
+
+        envs: Dict[str, str] = {}
+        # Add bundled packages to the PYTHONPATH. Note
+        # that this is necessary to achieve the same environment as Truss Server
+        # -- this is setup that is done by Truss Server, that won't be available
+        # to the standalone script.
+        bundled_packages_path = Path("/packages")
+        envs["PYTHONPATH"] = bundled_packages_path.as_posix()
+
+        return Docker.client().run(
+            image.id,
+            entrypoint="python",
+            command=["/app/script.py"],
+            detach=False,
+            mounts=[
+                [
+                    "type=bind",
+                    f"src={str(secrets_mount_dir_path)}",
+                    "target=/secrets",
+                ],
+                [
+                    "type=bind",
+                    f"src={str(LocalConfigHandler.bptr_data_resolution_dir_path())}",
+                    "target=/bptr",
+                ],
+                [
+                    "type=bind",
+                    f"src={str(script_path.absolute())}",
+                    "target=/app/script.py",
+                ],
+            ],
+            gpus="all" if self._spec.config.resources.use_gpu else None,
```

Comment:
should we check if a gpu is also available? what heppens if you run this without a GPU available?

Reply:
good point, i'm pretty sure this does not work if you do not have gpus